src/plugins/calculationTry1/MathService.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { createContext } from "react";
+
+
+export type THistoryFormulas = {
```

Comment:
actually no need do this like an object when it has only one property.
Or convert it to an interface and add couple of methods (add(), del(), get(), toString()|Symbol.toPrimitive) as well.

Reply:
Discussed about React-comparing changes variables in React.Context.